### PR TITLE
Few improvements

### PR DESCRIPTION
**Pull request by yalh76**
## Solution

- *Few improvements*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

File: conf/production.yaml [hunk 1]
```diff
@@ -426,11 +429,11 @@ import:
         # Examples:
         #   * https://api.github.com/repos/ytdl-org/youtube-dl/releases
         #   * https://api.github.com/repos/yt-dlp/yt-dlp/releases
-        url: 'https://api.github.com/repos/yt-dlp/yt-dlp/releases'
```
Comment (fflorent) [inline conf/production.yaml:429]: May I suggest to wait for feedback from this PR before?
https://github.com/Chocobozzz/PeerTube/pull/4706

 - youtube-dl currently is **very** slow (50kb/s), which makes it unusable
 - yt-dlp has an issue that makes which makes it unusable at all but only for the moment on PeerTube and a fix may exist soon.
   - we may either wait for a new patch version of PeerTube to be released
   - or we may patch PeerTube until the new release is here as soon as it is merged and until the new version is released

What do you think?

File: scripts/upgrade [hunk 2]
```diff
@@ -253,6 +260,17 @@ pushd "$final_path"
...
+#=================================================
+# PEERTUBE 4.0.0 UPGRADE MIGRATION SCRIPT
+#=================================================
+
+if ynh_compare_current_package_version --comparison lt --version 4.0.0~ynh1; then
+	ynh_script_progression --message="Running Peertube 4.0.0 migration script..."
+	pushd "$final_path"
+		ynh_exec_warn_less sudo -u $app env $ynh_node_load_PATH NODE_CONFIG_DIR="$final_path/config" NODE_ENV=production node dist/scripts/migrations/peertube-4.0.js
+	popd
+fi
```
Comment (fflorent) [inline scripts/upgrade:272]: I need to do some check of this… The migration script *may* need the service to be running.

**Comment by yalh76** [comment]
!testme

**Comment by fflorent** [comment]
LGTM (with delay, sorry)

**Pull request merged**